packages/sst/src/constructs/SsrSite.ts

```diff
@@ -1113,6 +1113,9 @@ function handler(event) {
           compress: true,
           cachePolicy: CachePolicy.CACHING_OPTIMIZED,
           responseHeadersPolicy: cdk?.responseHeadersPolicy,
+          functionAssociations: [
```

Comment:
This seems to work for now but  I'm not sure if passing a function to all behaviors is a good idea

Reply:
Thanks @barakcodes! I added a `clientCFFunctionInjection` field and moved the code in there.